cmd/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/joho/godotenv"
+	db "github.com/nikaydo/final/internal/db"
+	"github.com/nikaydo/final/internal/server"
+)
+
+func checkEnv(s string) string {
+	e := os.Getenv(s)
+	if e == "" {
+		log.Println(s + " not set")
+		return ""
+	}
+	log.Println(s+":", e)
+	return e
+}
+
+func main() {
+	err := godotenv.Load("internal/config/.env")
+	if err != nil {
+		log.Fatal("Error loading .env file")
```

Comment:
не теряй сообщение с ошибкой
```suggestion
		log.Fatal("Error loading .env file", err)
```